initialUI.py

```diff
@@ -22,72 +22,130 @@
     Image,
     ElevatedButton,
     ProgressRing,
+    ProgressBar,
 )
 
 
-# Write main app code here
-# def mainApp(UserControl):
+class InitialPage(UserControl):
+    def __init__(self):
+        super().__init__()
+        self.title = None
+        self.img = None
+        self.connection_status = None
+        self.btn_pair = None
+        self.rtn_display = None
+
+    def build(self):
+        self.title = Container(
+            content=Text(
+                "Transportaser",
+                size=70,
+                color=colors.WHITE,
+                weight="normal"),
+            margin=margin.only(top=50)
+        )
+
+        self.img = Image(
+            src=f"images/lightning_icon1.png",
+            width=200,
+            height=200,
+            fit="contain",
+        )
+
+        self.connection_status = Container(
+            content=Text(
+                "No connection",
+                size=40,
+                color=colors.RED_500,
+                weight="normal")
+        )
+
+        self.btn_pair = Container(
+            content=Text("Pair",
+                         size=25,
+                         color="#253439",
+                         bgcolor=colors.WHITE,
+                         weight="normal"),
+            bgcolor=colors.WHITE,
+            alignment=alignment.center,
+            width=140,
+            height=35,
+            border_radius=20,
+            on_click=self.btn_pair_clicked,
+            margin=margin.only(top=30),
+        )
+
+        self.rtn_display = Column(
+            controls=[
+                self.title,
+                self.img,
+                self.connection_status,
+                self.btn_pair
+            ],
+            alignment='center',
+            horizontal_alignment='center',
+        )
+
+        return self.rtn_display
+
+    def btn_pair_clicked(self, e):
```

Comment:
```suggestion
    def button_pair_clicked(self, e):
```
the more descriptive the better

Reply:
Fuck u ! NO!